frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart

```diff
@@ -31,14 +32,19 @@ class AppFlowyEditorPage extends StatefulWidget {
   State<AppFlowyEditorPage> createState() => _AppFlowyEditorPageState();
 }
 
+final List<CommandShortcutEvent> defaultCommandShortcutEvents = [
+  ...codeBlockCommands.map((e) => e.copyWith()).toList(),
+  ...standardCommandShortcutEvents.map((e) => e.copyWith()).toList(),
+];
```

Comment:
Why is this different from `commandShortcutEvents`?

Reply:
Here the `defaultCommandShortcutEvents` is a copy of the original values of `codeBlockCommands` and `standardCommandShortcuts`. This enables us to store their default values, which are used in the Reset To Default Functionality. 
The `commandShortcutEvents`, on the other hand, are subject to change because we call the `updateCommand()` on their children. Thus we can't use `commandShortcutEvents` again for resetting to default values.

Reply:
Can we then initialisere the other using the original one.

Reply:
Yes. I have now created the `commandShortcutEvents` first and then made copy of it in `defaultCommandShortcutEvents`.